tests/test_generators.py

```diff
@@ -0,0 +1,57 @@
+import pandas as pd
+
+from transformers import pipeline
+
+from adatest import generators
+
+
+class TestTransformers:
+    def test_smoke(self):
+        hf_model = pipeline("text-generation", model="EleutherAI/gpt-neo-125M")
```

Comment:
This test would be very handy for seeing if a particular model will work with the Transformers generator. Consider abstracting away the model="EleutherAI/gpt-neo-125M" bit so we can do several tests with a variety of models. I wonder if the OPT models will work? I couldn't get them to work with the current Transformers generator, but maybe I was using the API incorrectly

Reply:
I can make that a parameter.

Reply:
@Nking92 done.